keyboard_rewrite/test/test_helper.rb

```diff
@@ -0,0 +1,35 @@
+SRC_PATH = File.expand_path("../../src", __FILE__)
+DATA_PATH = File.join(File.expand_path(File.dirname(__FILE__)), "data")
+ENV["Y2DIR"] = SRC_PATH
+
+require "yast"
+require "yast/rspec"
+
+if ENV["COVERAGE"]
+  require "simplecov"
+  SimpleCov.start do
+    add_filter "/test/"
+  end
+
+  # for coverage we need to load all ruby files
+  Dir["#{SRC_PATH}/lib/**/**/*.rb"].each { |f| require_relative f }
+
+  # use coveralls for on-line code coverage reporting at Travis CI
+  if ENV["TRAVIS"]
+    require "coveralls"
+    SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+      SimpleCov::Formatter::HTMLFormatter,
+      Coveralls::SimpleCov::Formatter
+    ]
+  end
+end
+
+def given_layouts(layouts_to_return)
```

Comment:
I would suggest to move these helper methods to its own module and including them using the RSpec config method. See [this example](https://github.com/yast/yast-installation/blob/5ae24ff36f370f41b84bed5ae5f3107bc0552f87/test/test_helper.rb#L59).